### ISSUE TITLE
Format docs to show with --help correctly

### ISSUE THREAD

**Issue by jekwatt** [hgsc-project-managers/pm-utils#7]
Currently it prints out (not intended):

```usage: report_merged_qc_results.py [-h] recent_merge_report output_file

New TOPMed metrics be added to Exemplar LIMS merge report: - PF HQ Aligned Q20
Bases - Mean Insert Size (Library AVG) # corrected value - WGS HET SNP Q - WGS
HET SNP SENSITIVITY Update code that uses a single input file (example in
below) and generates an XLSX workbook with two sheets: - tab3 for TOPMed
Weekly Report, 'Production Metrics' (tab3) - tm_qc for TOPMed sample QC e.g. -
merge_report_with_new_topmed_metrics.xlsx Columns in TOPMed Weekly Report tab3
'Production Metrics': 1 External ID # extract from merge_name 2 Collection #
Study/Cohort 3 PF HQ Aligned Q20 Bases # new 4 Mean Insert Size (Library AVG)
# new 5 Mean Coverage (Raw) # new correct value 6 WGS HET SNP Q # new 7 WGS
HET SNP SENSITIVITY # new 8 Per 10 Coverage Bases 9 Per 20 Coverage Bases 10
Q20 Bases 11 Contamination % # convert Contamination Rate to Contamination %
12 Notes

positional arguments:
  recent_merge_report  *.xlsx, usually downloaded from Exemplar LIMS
  output_file          should end with .xlsx

optional arguments:
  -h, --help           show this help message and exit
```

It should print out something more readable:
```
New TOPMed metrics be added to Exemplar LIMS merge report:
- PF HQ Aligned Q20 Bases
- Mean Insert Size (Library AVG) # corrected value
- WGS HET SNP Q
- WGS HET SNP SENSITIVITY

Update code that uses a single input file (example in below) and
generates an XLSX workbook with two sheets:
- tab3 for TOPMed Weekly Report, 'Production Metrics' (tab3)
- tm_qc for TOPMed sample QC

e.g.
- merge_report_with_new_topmed_metrics.xlsx

Columns in TOPMed Weekly Report tab4 'Production Metrics':
1 External ID  # extract from merge_name
2 Collection  # Study/Cohort
3 PF HQ Aligned Q20 Bases # new
4 Mean Insert Size (Library AVG)  # new
5 Mean Coverage (Raw)  # new correct value
6 WGS HET SNP Q  # new
7 WGS HET SNP SENSITIVITY  # new
8 Per 10 Coverage Bases
9 Per 20 Coverage Bases
10 Q20 Bases
11 Contamination %  # convert Contamination Rate to Contamination %
12 Notes

positional arguments:
  recent_merge_report  *.xlsx, usually downloaded from Exemplar LIMS
  output_file          should end with .xlsx

optional arguments:
  -h, --help           show this help message and exit
```

**Comment by walkerh** [comment]
https://docs.python.org/3/library/argparse.html#formatter-class

**Comment by jekwatt** [comment]
Fixed by PR #8.
